### ISSUE TITLE
Unstructured suddenly can't parse scanned PDFs

### ISSUE THREAD

**Issue by eRuaro** [Unstructured-IO/unstructured#717]
I'm using `Unstructured` for parsing PDFs and have it installed through a docker dev container. It was working a few months ago but after I rebuilt the container for deployment, it suddenly breaks.

Here's my dockerfile:
```
FROM python:3.9-slim-buster

# Update package lists
RUN apt-get update && apt-get install ffmpeg libsm6 libxext6 gcc g++ git build-essential libpoppler-cpp-dev libmagic-dev pkg-config poppler-utils tesseract-ocr libtesseract-dev -y

# Make working directories
RUN  mkdir -p  /app
WORKDIR  /app

# Copy the requirements.txt file to the container
COPY requirements.txt .

# Install dependencies
RUN pip install --upgrade pip

RUN pip install torch torchvision torchaudio

RUN pip install unstructured-inference

RUN pip install -r requirements.txt

RUN pip install 'git+https://github.com/facebookresearch/detectron2.git@e2ce8dc#egg=detectron2'

# Copy the .env file to the container
COPY .env .

# Copy every file in the source folder to the created working directory
COPY  . .

# Expose the port that the application will run on
EXPOSE 8080

# Start the application
CMD ["python3.9", "-m", "uvicorn", "main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "8080", "--workers", "4"]
```

and `requirements.txt`:
```
fastapi
uvicorn
langchain
python-poppler
pytesseract
unstructured[local-inference]
psycopg2-binary
pgvector
openai
tiktoken
python-dotenv
pypdf
```

I've verified that detectron is in my system but it seems like it's not being used by unstructured as it defaults to using `pdfminer`

**Comment by MthwRobinson** [comment]
@eRuaro - have you tried passing in `strategy="hi_res"`? Prior to version `0.6.6`, `unstructured` defaulted to the `"hi_res"` strategy (which uses detectron2). After `0.6.6.` it chooses a strategy based on the kwargs and the document characteristics.

If I PDF doesn't have extractable text, it should default to `"hi_res"` or `"ocr_only"` and no to `"fast"` (which is the strategy that uses `pdfminer`). Could you run the following on your document and let us know what the result is?

```python
from unstructured.partition.strategies import is_pdf_text_extractable

is_pdf_text_extractable(filename="your-file.pdf")
```

**Comment by qued** [comment]
@eRuaro Did you try passing in the `strategy="hi_res"` parameter as suggested? I have a [PR](https://github.com/Unstructured-IO/unstructured/pull/900) in that I hope addresses the issue you are/were having, but it would be good to know if it's solving the same problem you are experiencing.

**Comment by qued** [comment]
Closed in view of #900 .
